.zenodo.json

```diff
@@ -0,0 +1,402 @@
+{
+  "creators": [
+    {
+      "name": "Luis Ibanez"
+    },
+    {
+      "name": "Bill Lorensen"
+    },
+    {
+      "affiliation": "Kitware, Inc",
+      "name": "McCormick, Matthew",
+      "orcid": "0000-0001-9475-3756"
+    },
+    {
+      "name": "Brad King"
+    },
+    {
+      "name": "Daniel Blezek"
+    },
+    {
+      "affiliation": "University of Iowa",
+      "name": "Johnson, Hans",
+      "orcid": "0000-0001-9513-2660"
+    },
+    {
+      "name": "Bradley Lowekamp"
+    },
+    {
+      "name": "Julien Jomier"
+    },
+    {
+      "name": "Jim Miller"
+    },
+    {
+      "name": "Ga\u00ebtan Lehmann"
+    },
+    {
+      "name": "Josh Cates"
+    },
+    {
+      "name": "Lydia Ng"
+    },
+    {
+      "name": "Jisung Kim"
+    },
+    {
+      "name": "Arnaud Gelas"
+    },
+    {
+      "name": "Mathieu Malaterre"
+    },
+    {
+      "name": "Karthik Krishnan"
+    },
+    {
+      "name": "Bill Hoffman"
+    },
+    {
+      "name": "Kent Williams"
+    },
+    {
+      "name": "Francois Budin"
+    },
+    {
+      "affiliation": "Universit\u00e9 de Sherbrooke",
+      "name": "Legarreta, Jon Haitz",
+      "orcid": "0000-0002-9661-1396"
+    },
+    {
+      "name": "Will Schroeder"
+    },
+    {
+      "name": "Stephen R. Aylward"
+    },
+    {
+      "name": "D\u017eenan Zuki\u0107"
+    },
+    {
+      "name": "Xiaoxiao Liu"
+    },
+    {
+      "name": "Brian Avants"
+    },
+    {
+      "affiliation": "Kitware, Inc",
+      "name": "McCormick, Matthew",
+      "orcid": "0000-0001-9475-3756"
+    },
+    {
+      "name": "Niels Dekker"
+    },
+    {
+      "name": "Aljaz Noe"
+    },
+    {
+      "name": "Michka Popoff"
+    },
+    {
+      "name": "Nick Tustison"
+    },
+    {
+      "name": "Gabe Hart"
+    },
+    {
+      "name": "D\u017eenan Zuki\u0107"
+    },
+    {
+      "name": "Sean McBride"
+    },
+    {
+      "name": "Tessa Sundaram"
+    },
+    {
+      "name": "Alexandre Gouaillard"
+    },
+    {
+      "name": "Michael Stauffer"
+    },
+    {
+      "name": "Andinet Enquobahrie"
+    },
+    {
+      "name": "Sayan Pathak"
+    },
+    {
+      "name": "Andy Cedilnik"
+    },
+    {
+      "name": "Ting Chen"
+    },
+    {
+      "name": "Damion Shelton"
+    },
+    {
+      "name": "Brian Helba"
+    },
+    {
+      "name": "Cory Quammen"
+    },
+    {
+      "affiliation": "Universit\u00e9 de Sherbrooke",
+      "name": "Legarreta, Jon Haitz",
+      "orcid": "0000-0002-9661-1396"
+    },
+    {
+      "name": "Yinpeng Jin"
+    },
+    {
+      "name": "Dirk Padfield"
+    },
+    {
+      "name": "Tom Vercauteren"
+    },
+    {
+      "name": "Hyun Jae Kang"
+    },
+    {
+      "name": "Matt Turek"
+    },
+    {
+      "name": "Robert Tamburo"
+    },
+    {
+      "name": "Michel Audette"
+    },
+    {
+      "name": "Mark Foskey"
+    },
+    {
+      "name": "Pablo Hernandez-Cerdan"
```

Comment:
```suggestion
      "name": "Hernandez-Cerdan, Pablo"
      "orcid":  "0000-0002-1689-3451"
```

Reply:
Good catch -- the commas were missing from the initial addition, and they were missing from the script. Added in the latest push.